src/Controllers/Admin/AbstractAdminController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gett\MyparcelBE\Controllers\Admin;
```

Comment:
Kunnen we de namespace niet nog even van onszelf maken in plaats van van `Gett`?

Reply:
Soon™️ :)